src/foundry/client/data/documents/setting.d.mts

```diff
@@ -13,13 +13,22 @@ declare global {
    * @see {@link WorldSettings}       The world-level collection of Setting documents
    */
   class Setting extends ClientDocumentMixin(foundry.documents.BaseSetting) {
+    static #PRIMITIVE_TYPES: readonly [
```

Comment:
was left out previously due to being private member, do we otherwise need this for downstream types?

Reply:
Note: if a class has a truly private member we should keep around at least one version. It doesn't have to match runtime; no one can access it, not even subclasses etc. So it can just be something simple like `#setting: true` that just indicates there _is_ a private property.

So to the type system there's no difference between 1 truly private property and 10,000 (assuming they don't effect variance) but there is a difference between 0 and 1:
```ts
declare class Foo {
    foo: number
}

// Ok by TS's rules.
const foo: Foo = { foo: 1 }

declare class Bar {
    #trulyPrivateProperty: "exists"
    bar: number
}

// NOT okay by TS's rules.
const bar: Bar = { bar: 1 }
```

Reply:
Ah that's important to know. Will need to get re-audited because I've been getting that wrong.